src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,102 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    private final By allSongs = By.cssSelector("a[href='#!/songs']");
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+
+    public void enterAllSongs() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
+        WebElement songs = wait.until(ExpectedConditions.visibilityOfElementLocated(allSongs));
+        songs.click();
+
+    }
+
+
+   // public HomePage(WebDriver givenDriver) {
+       // super(givenDriver);
+    //}
+
+    //public void enterAllSongs() {
+
+        //WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
+        //WebElement allSongs = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a[href='#!/songs']")));
+        //allSongs.click();
+
+        // WebElement allSongs = driver.findElement(By.cssSelector("a[href='#!/songs']"));
+        //allSongs.click();
+
+        //Thread.sleep(1000);
+        // allSongs.click();
+        //
+    }
+
+    public void selectSong() {
+        WebElement song = driver.findElement(By.xpath("//tr[@class='song-item']"));
+        song.click();
+
+
+    }
+
+    public void enterButtonPlaySong() {
+        WebElement buttonPlaySong = driver.findElement(By.xpath("//span[@title='Play or resume']"));
+        Actions actions = new Actions(driver);
+        actions.click(buttonPlaySong).perform();
+    }
+
+
+    public boolean isDisplayedPlayingSong() {
+
+        WebElement songIsPlaying = driver.findElement(By.cssSelector("[data-testid = 'sound-bar-play']"));
+        return songIsPlaying.isDisplayed();
+
```

Comment:
For example

  private final By allSongs = By.cssSelector("a[href='#!/songs']");

    public HomePage(WebDriver givenDriver) {
        super(givenDriver);
    }


    public void enterAllSongs() {
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(4));
        WebElement songs = wait.until(ExpectedConditions.visibilityOfElementLocated(allSongs));
        songs.click();

    }